doc/source/structures/vessels/attitudecontroller.rst

```diff
@@ -0,0 +1,245 @@
+.. _attitudecontroller:
+
+Attitude Controller
+======
+
+A ship usually has various attitude controllers like control surfaces, engines, RCS thrusters, rotors and drain valves. These controllers can be configured as follows:
+
+    local controllers to ship:AttitudeControllers.
+    print(controllers[0]:ControllerType + " allows pitch: " + controllers[0]:allowPitch).
+
+.. structure:: AttitudeController
+
+    .. list-table:: Members
+        :header-rows: 1
+        :widths: 1 1 2
+
+        * - Suffix
+          - Type (units)
+          - Description
+
+        * - :attr:`PART`
+          - :struct:`Part <Part>`
+          - The part this controller belongs to.
+        * - :attr:`MODULE`
+          - :struct:`PartModule <PartModule>`
+          - The module this controller belongs to. Will return false if there is no matching module.
+        * - :attr:`ALLOWPITCH`
+          - :ref:`Boolean <boolean>`
+          - Gets or sets wheter this controller should respond to pitch input.
+        * - :attr:`ALLOWYAW`
+          - :ref:`Boolean <boolean>`
+          - Gets or sets wheter this controller should respond to yaw input.
+        * - :attr:`ALLOWROLL`
+          - :ref:`Boolean <boolean>`
+          - Gets or sets wheter this controller should respond to roll input.
+        * - :attr:`ALLOWX`
+          - :ref:`Boolean <boolean>`
+          - Gets or sets wheter this controller should respond to x translation input.
+        * - :attr:`ALLOWY`
+          - :ref:`Boolean <boolean>`
+          - Gets or sets wheter this controller should respond to y translation input.
+        * - :attr:`ALLOWZ`
+          - :ref:`Boolean <boolean>`
+          - Gets or sets wheter this controller should respond to z translation input.
+        * - :attr:`HASCUSTOMTHROTTLE`
+          - :ref:`Boolean <boolean>`
+          - Wheter this controller has a custom throttle input.
+        * - :attr:`CUSTOMTHROTTLE`
+          - :ref:`scalar <scalar>` (%)
+          - The value the custom throttle.
+        * - :attr:`ROTATIONAUTHRORITYLIMITER`
```

Comment:
Typo "AuthRORity" -> Authority".

Reply:
This typo is also in the actual code, so I was having trouble testing this, trying to print the ROTATIONAUTHORITY and it kept saying there's no such suffix which confused me until I carefully noticed the spelling.